config/api/src/main/java/org/jboss/windup/config/AbstractRuleProvider.java

```diff
@@ -131,7 +133,7 @@ public String toString()
         StringBuilder builder = new StringBuilder();
         builder.append(metadata.getID());
 
-        if (!metadata.getID().equals(metadata.getOrigin()))
+        if (this instanceof RulePhase || !metadata.getID().equals(metadata.getOrigin()))
```

Comment:
If it a RulePhase, this would evaluate to true, thus causing the origin to be printed. This doesn't seem to relate to the ID at all?